projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/notification/EmailPropertiesConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.notification;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class EmailPropertiesConfiguration {
```

Comment:
nit: Properties and Configuration are generally the same thing (broadly) or very close. Just "EmailConfiguration" seems good enough ?